appcues/src/main/java/com/appcues/ui/ExperienceStepFormState.kt

```diff
@@ -19,11 +19,11 @@ internal class ExperienceStepFormState {
     val isFormComplete = mutableStateOf(true)
 
     fun getValue(primitive: TextInputPrimitive) =
-        getItem(primitive).value
+        getItem(primitive).text
```

Comment:
the form state now just returns mutableState values that the Composables directly bind to